eZ/Publish/Core/FieldType/Image/ImageStorage.php

```diff
@@ -113,6 +113,10 @@ public function storeFieldData(VersionInfo $versionInfo, Field $field, array $co
                 return false;
             }
 
+            if (!isset($field->value->data['id'])) {
```

Comment:
I still feel that actual bug is somewhere [here](https://github.com/ezsystems/ezpublish-kernel/blob/6.7/eZ/Publish/Core/FieldType/Image/ImageStorage.php#L109) . @alongosz is this ok that `$field->value->data` is not `null` when image is empty?